Javadoc-related build failure

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.10.3:jar (attach-javadocs) on project simplified-app-openebooks: MavenReportException: Error while generating Javadoc:
[ERROR] Exit code: 1 - /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:95: error: unknown tag: colgroup
[ERROR] <colgroup align="left" />
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:96: error: unknown tag: colgroup
[ERROR] <colgroup align="left" />
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:97: error: unknown tag: colgroup
[ERROR] <colgroup align="left" />
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:102: error: no summary or caption for table
[ERROR] </table>
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:152: error: unknown tag: colgroup
[ERROR] <colgroup align="left" />
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:153: error: unknown tag: colgroup
[ERROR] <colgroup align="left" />
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:154: error: unknown tag: colgroup
[ERROR] <colgroup align="left" />
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:158: error: no summary or caption for table
[ERROR] </table>
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1257: error: unknown tag: colgroup
[ERROR] <colgroup align="left" />
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1258: error: unknown tag: colgroup
[ERROR] <colgroup align="left" />
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1269: error: no summary or caption for table
[ERROR] </table>
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1287: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1291: error: unknown tag: attr
[ERROR] @attr name android:background
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1296: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1300: error: unknown tag: attr
[ERROR] @attr name android:orientation
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1305: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1316: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:centered
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1321: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1333: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:fillColor
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1338: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1350: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:pageColor
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1355: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1368: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:radius
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1373: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1384: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:snap
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1389: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1401: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:strokeColor
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1406: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1419: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:strokeWidth
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1425: error: unknown tag: colgroup
[ERROR] <colgroup align="left" />
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1426: error: unknown tag: colgroup
[ERROR] <colgroup align="left" />
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1435: error: no summary or caption for table
[ERROR] </table>
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1450: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1454: error: unknown tag: attr
[ERROR] @attr name android:background
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1459: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1470: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:centered
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1475: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1488: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:gapWidth
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1493: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1506: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:lineWidth
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1511: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1523: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:selectedColor
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1528: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1541: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:strokeWidth
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1546: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1558: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:unselectedColor
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1564: error: unknown tag: colgroup
[ERROR] <colgroup align="left" />
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1565: error: unknown tag: colgroup
[ERROR] <colgroup align="left" />
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1582: error: no summary or caption for table
[ERROR] </table>
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1607: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1611: error: unknown tag: attr
[ERROR] @attr name android:background
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1616: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1620: error: unknown tag: attr
[ERROR] @attr name android:textColor
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1625: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1629: error: unknown tag: attr
[ERROR] @attr name android:textSize
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1634: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1647: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:clipPadding
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1652: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1664: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:footerColor
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1669: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1682: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:footerIndicatorHeight
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1687: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1693: error: unknown tag: colgroup
[ERROR] <colgroup align="left" />
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1694: error: unknown tag: colgroup
[ERROR] <colgroup align="left" />
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1695: error: unknown tag: colgroup
[ERROR] <colgroup align="left" />
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1700: error: no summary or caption for table
[ERROR] </table>
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1702: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:footerIndicatorStyle
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1707: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1720: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:footerIndicatorUnderlinePadding
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1725: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1738: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:footerLineHeight
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1743: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1756: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:footerPadding
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1761: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1767: error: unknown tag: colgroup
[ERROR] <colgroup align="left" />
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1768: error: unknown tag: colgroup
[ERROR] <colgroup align="left" />
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1769: error: unknown tag: colgroup
[ERROR] <colgroup align="left" />
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1773: error: no summary or caption for table
[ERROR] </table>
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1775: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:linePosition
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1780: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1791: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:selectedBold
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1796: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1808: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:selectedColor
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1813: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1826: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:titlePadding
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1831: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1844: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:topPadding
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1850: error: unknown tag: colgroup
[ERROR] <colgroup align="left" />
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1851: error: unknown tag: colgroup
[ERROR] <colgroup align="left" />
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1858: error: no summary or caption for table
[ERROR] </table>
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1871: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1875: error: unknown tag: attr
[ERROR] @attr name android:background
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1880: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1891: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:fadeDelay
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1896: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1907: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:fadeLength
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1912: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1923: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:fades
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1928: error: unknown tag: attr
[ERROR] @attr description
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/generated-sources/r/org/nypl/labs/OpenEbooks/app/R.java:1940: error: unknown tag: attr
[ERROR] @attr name org.nypl.labs.OpenEbooks.app:selectedColor
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/unpacked-libs/cv_library_2.4.1/src/com/viewpagerindicator/PageIndicator.java:30: warning: no description for @param
[ERROR] * @param view
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/unpacked-libs/cv_library_2.4.1/src/com/viewpagerindicator/PageIndicator.java:37: warning: no description for @param
[ERROR] * @param view
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/unpacked-libs/cv_library_2.4.1/src/com/viewpagerindicator/PageIndicator.java:38: warning: no description for @param
[ERROR] * @param initialPosition
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/unpacked-libs/cv_library_2.4.1/src/com/viewpagerindicator/PageIndicator.java:48: warning: no description for @param
[ERROR] * @param item
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/unpacked-libs/cv_library_2.4.1/src/com/viewpagerindicator/PageIndicator.java:55: warning: no description for @param
[ERROR] * @param listener
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/unpacked-libs/cv_library_2.4.1/src/com/viewpagerindicator/IconPagerAdapter.java:7: warning: no @param for index
[ERROR] int getIconResId(int index);
[ERROR] ^
[ERROR] /Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/unpacked-libs/cv_library_2.4.1/src/com/viewpagerindicator/IconPagerAdapter.java:7: warning: no @return
[ERROR] int getIconResId(int index);
[ERROR] ^
[ERROR] 
[ERROR] Command line was: /Library/Java/JavaVirtualMachines/jdk1.8.0_74.jdk/Contents/Home/jre/../bin/javadoc @options @packages
[ERROR] 
[ERROR] Refer to the generated Javadoc files in '/Users/aferditamuriqi/IdeaProjects/android/simplified-app-openebooks/target/apidocs' dir.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :simplified-app-openebooks
```

--- a/pom.xml
+++ b/pom.xml
@@ -539,6 +539,10 @@
                   <Built-By>NYPL Labs</Built-By>
                 </manifestEntries>
               </archive>
+              <sourcepath>${basedir}/target/generated-sources</sourcepath>
+              <sourceFileExcludes>
+                <exclude>**/R.java</exclude>
+              </sourceFileExcludes>
             </configuration>
           </execution>
         </executions>

